src/main/java/kr/codesquad/domain/Line.java

```diff
@@ -0,0 +1,33 @@
+package kr.codesquad.domain;
+
+import java.util.List;
+
+public class Line {
+
+    private final RandomGenerator randomGenerator = new RandomGenerator();
+    private final List<Boolean> points;
+
+    public Line(List<String> people) {
+        List<Boolean> randomBooleans = randomGenerator.generate(people.size() - 1);
```

Comment:
```suggestion
        List<Boolean> randomBooleans = randomGenerator.generateRandomBooleans(people.size() - 1);
```

지금도 무난하지만 조금 더 구첵적인 이름은 어때요?